contracts/CryptoChampions.sol

```diff
@@ -388,32 +385,32 @@ contract CryptoChampions is ICryptoChampions, AccessControl, ERC1155, VRFConsume
         (_heroes[heroId].weather, newRandomNumber) = _rollDice(5, newRandomNumber); // 1 ouf of 5
 
         (_heroes[heroId].hp, newRandomNumber) = _rollDice(21, newRandomNumber); // Roll 10-30
-        _heroes[heroId].hp = _heroes[heroId].hp.add(9);
+        _heroes[heroId].hp = _heroes[heroId].hp + (9);
```

Comment:
Any reason why you are no longer using SafeMath here?  Is it because of gas fees?

Reply:
It shaved off a little bit of gas consumption, but now the function fits even with safemath so ill put it back.